.travis.yml

```diff
@@ -19,11 +19,6 @@ matrix:
 #      env:
 #        - PYTHON_VERSION=3.4.5
```

Comment:
That would make sense, the idea would be to submit a recipe to conda-forge. With that in place, the package would be available on conda on linux, macosx and windows. 

That said, conda-forge recipe are updated only when there is a release of the package. If you would like to setup continuous integration using miniconda, you would have to set it up on your own.

I currently do not have the bandwidth to set miniconda CI up for pyradiomics, in the mean time I would be happy to review PR.

@blowekamp Do we know when macosx wheels will be available for python 35 ?